charts/multicluster-scheduler/templates/cm.yaml

```diff
@@ -4,31 +4,5 @@ metadata:
   name: {{ include "fullname" . }}
   labels: {{ include "labels" . | nindent 4 }}
 data:
-  proxy-scheduler-config: |
-    apiVersion: kubescheduler.config.k8s.io/v1beta3
-    kind: KubeSchedulerConfiguration
-    leaderElection:
-      leaderElect: true
-      resourceName: admiralty-proxy-scheduler
-      resourceNamespace: {{ .Release.Namespace }}
-      resourceLock: leases
-    profiles:
-      - schedulerName: admiralty-proxy
-        plugins:
-          multiPoint:
-            enabled:
-              - name: proxy
-  candidate-scheduler-config: |
-    apiVersion: kubescheduler.config.k8s.io/v1beta3
-    kind: KubeSchedulerConfiguration
-    leaderElection:
-      leaderElect: true
-      resourceName: admiralty-candidate-scheduler
-      resourceNamespace: {{ .Release.Namespace }}
-      resourceLock: leases
-    profiles:
-      - schedulerName: admiralty-candidate
-        plugins:
-          multiPoint:
-            enabled:
-              - name: candidate
+  proxy-scheduler-config: {{ .Values.scheduler.proxySchedulerConfigYaml | toYaml | indent 1 }}
```

Comment:
Last I checked, Helm values don't support templating, so `{{ .Release.Namespace }}` wouldn't expand. You'd have to use the [tpl](https://helm.sh/docs/howto/charts_tips_and_tricks/#using-the-tpl-function) function.

Also, why `indent 1`.

Reply:
good catch, re the template var. I've fixed it in the latest commit. 

Also the indent is needed, otherwise the rendered YAML is invalid:

```
apiVersion: v1
kind: ConfigMap
data:
  proxy-scheduler-config: |-
  apiVersion: kubescheduler.config.k8s.io/v1beta3
  kind: KubeSchedulerConfiguration
  leaderElection:
    leaderElect: true
    resourceName: admiralty-proxy-scheduler
    resourceNamespace: admiralty
    resourceLock: leases
  profiles:
    - schedulerName: admiralty-proxy
      plugins:
        multiPoint:
          enabled:
            - name: proxy
  candidate-scheduler-config: |-
  apiVersion: kubescheduler.config.k8s.io/v1beta3
  kind: KubeSchedulerConfiguration
  leaderElection:
    leaderElect: true
    resourceName: admiralty-candidate-scheduler
    resourceNamespace: admiralty
    resourceLock: leases
  profiles:
    - schedulerName: admiralty-candidate
      plugins:
        multiPoint:
          enabled:
            - name: candidate
```